src/objecttypes/accounts/apps.py

```diff
@@ -1,5 +1,26 @@
-from django.apps import AppConfig
+from django.apps import AppConfig, apps
+from django.conf import settings
+from django.contrib.contenttypes.management import create_contenttypes
+from django.core.management import call_command
+from django.db.models.signals import post_migrate
+
+
+def update_admin_index(sender, **kwargs):
+    from django_admin_index.models import AppGroup
+
+    AppGroup.objects.all().delete()
+
+    # Make sure Objecttypes models are registered.
+    for app in settings.INSTALLED_APPS:
+        if app.startswith("objecttypes"):
+            app_config = apps.get_app_config(app.split(".")[-1])
+            create_contenttypes(app_config)
+
+    call_command("loaddata", "default_admin_index", verbosity=0)
```

Comment:
Do we really need to rewrite admin groups every time after migration? Since they are not changed very often we can have just an additional data migration?

Reply:
In theory, I agree. But I see what happens in Open Zaak and the admin index is already out of date because data migrations are a pain to write for just a menu.

This hook makes it so much easier to maintain an up-to-date menu as it is intended. Just update the fixtures, commit and done.

Reply:
Yeah. the solution with admin fixtures in Openzaak proved not to be optimal